app/presenters/multi_volume_work_show_presenter.rb

```diff
@@ -1,9 +1,20 @@
 class MultiVolumeWorkShowPresenter < CurationConcernsShowPresenter
   def file_presenter_class
-    ::ScannedResourceShowPresenter
+    DynamicShowPresenter.new
   end
 
   def viewing_hint
     'multi-part'
   end
+
+  class DynamicShowPresenter
+    def new(*args)
```

Comment:
Is there any reason why the body of this method can't just go in `file_presenter_class` above?